### PR TITLE
Avoid writing vendor prefixes

### PR DESCRIPTION
**Pull request by tysongach**
I’d like to encourage use of Autoprefixer. It might be a bit wordy.

Thoughts?

### CODE REVIEW

**Comment by mike-burns** [comment]
This seems like a best practice.

**Comment by tysongach** [comment]
Great point @mike-burns! Updated.

**Comment by brendastorer** [comment]
Yes yes yes!!

**Comment by joshuaogle** [comment]
Autoprefixer doesn't handle properties that aren't supported across browsers, like `-webkit-font-smoothing`, `-webkit-tap-highlight-color` and things like `::-webkit-scrollbar`. We rarely/never write those, but maybe "User Autoprefixer..." would be better than "Avoid ... and use ..."?

**Comment by jasonramirez** [comment]
+1 to what Mr. Ogle says.

**Comment by tysongach** [comment]
Updated.

**Comment by joshuaogle** [comment]
Nice :musical_note:
